lib/active_fedora/relation/delegation.rb

```diff
@@ -7,7 +7,7 @@ module Delegation # :nodoc:
     # may vary depending on the klass of a relation, so we create a subclass of Relation
     # for each different klass, and the delegations are compiled into that subclass only.
 
-    BLACKLISTED_ARRAY_METHODS = [
```

Comment:
As this was cloned from ActiveRecord, this might be a better change: https://github.com/rails/rails/commit/9d79334a1dee67e31222c790e231772deafcaeb8